build/packaging/srpm/condor.spec

```
@@ -601,6 +593,21 @@ BOSCO provides an overlay system so the remote clusters appear to be a HTCondor
 cluster.  This allows the user to run their workflows using HTCondor tools across
 multiple clusters.
 
+#######################
+%package blahp
+Summary: BLAHP daemon
+Url: https://github.com/htcondor/BLAH
+Group: System/Libraries
+BuildRequires:  docbook-style-xsl, libxslt
+%if 0%{?rhel} >= 8 || 0%{?fedora}
+Requires: python3
+%else
+Requires: python >= 2.2
+%endif
+
```

Need to say that condor-blahp replaces blahp
```suggestion
Provides: blahp = %{version}-%{release}
Obsoletes: blahp < 9.5.0

```